bin/improver-percentiles-to-probabilities

```diff
@@ -78,17 +78,17 @@ def main():
                         "calculated.")
     parser.add_argument("output_filepath", metavar="OUTPUT_FILE",
                         help="The output path for the processed NetCDF")
-    parser.add_argument("--new_name", metavar="NEW_NAME", default=None,
-                        type=str,
-                        help="Name for data in output file.  Defaults to "
-                        "'probability_of_X', where X is the name of the "
-                        "percentiled diagnostic.")
+    parser.add_argument("output_diagnostic_name",
+                        metavar="OUTPUT_DIAGNOSTIC_NAME", type=str,
+                        help="Name for data in output file e.g. "
+                             "probability_of_snowfall")
```

Comment:
This docstring needs to conform to the IMPROVER standard, which was agreed as eg "probability_of_snow_falling_level_below_ground_level".

Reply:
Done.